runtime/oti/jvminit.h

```diff
@@ -542,6 +542,8 @@ enum INIT_STAGE {
 #define VMOPT_XXDYNAMICHEAPIFICATION "-XX:+DynamicHeapification"
 #define VMOPT_XXNODYNAMICHEAPIFICATION "-XX:-DynamicHeapification"
 
+#define VMOPT_XXCONTINUATIONCACHE "-XX:continuationcache:"
```

Comment:
```suggestion
#define VMOPT_XXCONTINUATIONCACHE "-XX:ContinuationCache:"
```

We typically use capital letters to start -XX options and to delineate words

Reply:
fixed